pages/linux/umask.md

```diff
@@ -0,0 +1,15 @@
+# umask
+
+> Display file creation mask or set file creation mask to octal value.
+
+- Display current value as octal:
+
+`umask`
+
+- Display current value symbolically:
+
+`umask -S`
+
+- Turn off permission bits with octal value 'nnnn':
```

Comment:
Huh. I didn't get this at first, but re-reading [Wikipedia](https://en.wikipedia.org/wiki/Umask) (emphasis mine):

> The mask **restricts** permission settings. If the mask has a bit set to "1", it means the corresponding initial file permission will be **disabled**. A bit set to "0" in the mask means that the corresponding permission will be determined by the program and the system. In other words, the mask acts as a last-stage filter that **strips away permissions** as a file is created; **each bit that is set to a "1" strips away its corresponding permission**.

This isn't clear in the command description. We should find a way to make that clear, without making it too long.

Also, perhaps we should add at least one example using the [symbolic notation](https://en.wikipedia.org/wiki/Umask#Setting_the_mask_using_symbolic_notation), which is more intuitive.